examples/src/providers/rule_closures/conditions.js

```diff
@@ -48,7 +74,12 @@ module.exports = [
         // - query: (object) - An object that has all the query string items.
         // - params?: (optional Array<string>) - This is the PATH of the request.
         // Ensure the request: is a POST + has a BODY
-        return !(facts.httpRequest && facts.httpRequest.body && facts.httpRequest.method && facts.httpRequest.method === 'POST');
+        return !(
+          facts.httpRequest &&
+          facts.httpRequest.body &&
+          facts.httpRequest.method &&
+          facts.httpRequest.method === 'POST'
+        );
```

Comment:
not critical, but can be shortened with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

```suggestion
        return !(
          facts.httpRequest?.body &&
          facts.httpRequest?.method === 'POST'
        );
```

Reply:
Good call!